### PR TITLE
Expose fs and sampleRate for value subscription

### PR DESCRIPTION
**Pull request by nuustik**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -515,6 +508,16 @@ studio.internal = (function(proto) {
       app.makeSetterRequest(id, lastInfo.value_type, value, timestamp);
       //when offline must queue or update pending set request and call set callbacks ...???
     };
+
+    this.async._makeGetterRequest = function() {
+      if (valueSubscriptions.length > 0) {
+        var maxFs = Math.max.apply(Math, valueSubscriptions.map(v => v[1]));
+        var maxSampleRate = Math.max.apply(Math, valueSubscriptions.map(v => v[2]));
```
Comment (Karmo7) [inline index.js:515]: Technically sample_rate of 0 is the highest sample rate, so Math.max is not sufficient.

**Review by Karmo7 — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (nuustik) [reply index.js:515]: was approved in python client

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cdp-client",
-  "version": "2.0.1",
+  "version": "2.0.2",
```
[the remark this answers is not in the document]
Comment (nuustik) [reply package.json:3]: was approved in python client

File: index.js (hunk 1 above)
Comment (Karmo7) [reply index.js:515]: Should be fixed there as well then

**Pull request merged**